crates/forge/tests/it/test_helpers.rs

```diff
@@ -247,6 +247,7 @@ impl ForgeTestData {
     pub fn new(profile: ForgeTestProfile) -> Self {
         init_tracing();
 
+        let _ = rustls::crypto::ring::default_provider().install_default();
```

Comment:
Please add a comment why we have this

```suggestion
        // NOTE(zk): We need to manually install the crypto provider as zksync-era uses `aws-lc-rs` provider, while
        // foundry uses the `ring` provider. As a result, rustls cannot disambiguate between the two while selecting a default provider.
        let _ = rustls::crypto::ring::default_provider().install_default();
```